src/main/resources/crafter/core/core.properties

```diff
@@ -132,10 +132,12 @@ crafter.core.rest.views.json.prefixJson=false
 crafter.core.rest.views.json.renderSingleAttributeAsRootObject=true
 # True to render attributes as properties of the JSON object, false to omit them.
 crafter.core.rest.views.json.renderAttributes=false
+# The tree depth limit. This value must not be greater than the value of TREE_DEPTH_HARD_LIMIT=20
+crafter.core.rest.content.store.tree.depth.limit=10
 # The content store URLs (comma-separated list) that are allowed to be accessed through the REST API
-crafter.core.rest.content.store.url.allowedPatterns=
+crafter.core.rest.content.store.url.allowedPatterns=^/?site(/.*)?$
 # The content store URLs (comma-separated list) that are forbidden to be accessed through the REST API
-crafter.core.rest.content.store.url.forbiddenPatterns=^/?config(/.+)?$
+crafter.core.rest.content.store.url.forbiddenPatterns=
```

Comment:
Why are we clearing this property?

Reply:
```
private boolean isUrlAllowed(String url) {
        return (ArrayUtils.isEmpty(allowedUrlPatterns) || RegexUtils.matchesAny(url, allowedUrlPatterns)) &&
               (ArrayUtils.isEmpty(forbiddenUrlPatterns) || !RegexUtils.matchesAny(url, forbiddenUrlPatterns));
    }
```

So if we only allowed `/site/**` that means any other locations including `/config/**` are not allowed. Adding this to forbidden patterns won't be necessary in this case.

Reply:
But isn't it better to keep the forbidden by default in case someone wanted to open things up again, they won't open this sensitive area?

Reply:
You are right. I rolled back this change.